<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetThresholdDialog</class>
 <widget class="QDialog" name="SetThresholdDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set threshold</string>
  </property>
  <widget class="QCheckBox" name="checkBoxModeOneWork">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>511</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>同一ワーク内でのヒストグラム</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxModeMultiWork">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>501</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>複数ワークでの同一個所ヒストグラム</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetAlgorithm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>311</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Use</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Algorithm</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButtonSet">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>しきい値設定</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxAdoptRate">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="value">
    <double>80.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>採用率（％）</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxSigmaMultiply">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>20.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>6.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>設定する偏差値の倍率</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetHistogram">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>311</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Use</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Histogram</string>
    </property>
   </column>
  </widget>
  <widget class="QCheckBox" name="checkBoxCheckAllAlgorithm">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>108</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>All</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxCheckAllHistogram">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>108</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>All</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
